packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -126,6 +126,10 @@ export class ImportWallet extends React.Component<Props, State> {
     }
   }
 
+  isBackupPhraseValid() {
```

Comment:
Guess this should be `isBackupPhraseInvalid`

Reply:
Fixed! And added more details on _tested_ section. Is it ok?

Reply:
Is it fixed? Still shows as Valid for me